javers-core/src/main/java/org/javers/core/graph/ObjectGraphBuilder.java

```diff
@@ -156,15 +157,24 @@ private boolean isItemManagedType(ContainerType containerType){
      * is Map with ManagedClass on Key or Value position
      */
     private boolean isMapWithManagedTypes(EnumerableType enumerableType) {
-        if (! (enumerableType instanceof MapType)) {
+        if (enumerableType instanceof MapType){
```

Comment:
nie kopiujemy kodu, btw po co else if jeśli te bloki są identyczne?
jeśli zrobisz,np, że `MultimapTyp extends MapType` to nic nie trzeba tu kopiować